commands/cocktail-lookup.js

```diff
@@ -78,15 +92,15 @@ exports.cocktail = {
 					msg.channel.send({
 						embed: {
 							color: GReYBot.Config.defaultEmbedColor,
-							description: `${response.data.drinks[0].strDrink} is a good drink, but I don\'t have a good way to describe it.`
+							description: `${response.data.drinks[0].strDrink} is a good drink, but I don't have a good way to describe it.`
 						}
 					});
 				}
 			} else {
 				msg.channel.send({
 					embed: {
 						color: GReYBot.Config.defaultEmbedColor,
-						description: `Damn, I've never heard of that.  Where do I need to go to find it?`
```

Comment:
i still dont get how there were so many spacing issues... even after re-checking out the branch